pkg/client/kubernetes/apply.go

```diff
@@ -20,6 +20,8 @@ import (
 	"fmt"
 	"io"
 
+	"github.com/gardener/gardener/pkg/utils/errors"
```

Comment:
```suggestion
	utilerrors "github.com/gardener/gardener/pkg/utils/errors"
```
to not accidentally mix up with the core `errors` package